### PR TITLE
Bring back the parameters

### PR DESCRIPTION
**Pull request by YanisBft**
Brings back the parameters dropped when matching the snapshot, especially in particle classes

### CODE REVIEW

**Review by liach — APPROVED** [review]

File: mappings/net/minecraft/block/Block.mapping [hunk 1]
```diff
@@ -23,6 +23,8 @@ CLASS net/minecraft/class_2248 net/minecraft/block/Block
 		ARG 3 side
 	METHOD method_23349 getVelocityMultiplier ()F
 	METHOD method_23350 getJumpVelocityMultiplier ()F
+	METHOD method_27839 isBlock (Lnet/minecraft/class_2248;)Z
+		ARG 1 block
```
Comment (Juuxel) [inline mappings/net/minecraft/block/Block.mapping:27]: This seems like a weird name... What does an `isBlock(Block)` method do?

Comment (YanisBft) [reply mappings/net/minecraft/block/Block.mapping:27]: Checks if the block equals the block in parameter. Should I rename it to `equals`?

Comment (liach) [reply mappings/net/minecraft/block/Block.mapping:27]: why can't they use `==`?

Comment (liach) [reply mappings/net/minecraft/block/Block.mapping:27]: Yeah they just uses `this == block` in implementation. I guess this is their effort to move away from hardcoding; and I personally suggest just calling the method `is`.

**Review by Runemoro — APPROVED** [review]

File: mappings/net/minecraft/block/Block.mapping [hunk 2]
```diff
@@ -23,6 +23,8 @@ CLASS net/minecraft/class_2248 net/minecraft/block/Block
 		ARG 3 side
 	METHOD method_23349 getVelocityMultiplier ()F
 	METHOD method_23350 getJumpVelocityMultiplier ()F
+	METHOD method_27839 isBlock (Lnet/minecraft/class_2248;)Z
```
Comment (liach) [inline mappings/net/minecraft/block/Block.mapping]:
```suggestion
	METHOD method_27839 is (Lnet/minecraft/class_2248;)Z
```

File: mappings/net/minecraft/block/AbstractBlock.mapping [hunk 3]
```diff
@@ -522,6 +522,8 @@ CLASS net/minecraft/class_4970 net/minecraft/block/AbstractBlock
 		METHOD method_26234 isFullCube (Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;)Z
 			ARG 1 world
 			ARG 2 pos
+		METHOD method_27852 isBlock (Lnet/minecraft/class_2248;)Z
```
Comment (liach) [inline mappings/net/minecraft/block/AbstractBlock.mapping]:
```suggestion
		METHOD method_27852 is (Lnet/minecraft/class_2248;)Z
```

**Review by liach — COMMENTED** [review]
Optional name simplification.

**Review by Juuxel — CHANGES_REQUESTED** [review]
The two `isBlock` methods, `AbstractBlock.method_27852` and `Block.method_27839`, should have different Yarn names since they're different by their intermediary names.

**Comment by liach** [comment]
@Juuxel `method_27852` is in `AbstractBlockState`, not `AbstractBlock`

**Comment by Juuxel** [comment]
Oh, nevermind then. `isBlock` is still a bad name, though.

**Review by liach — APPROVED** [review]

**Review by Juuxel — APPROVED** [review]

**Comment by liach** [comment]
Just making sure that modmuss gets a ping on this ready-to-merge pr.

**Pull request merged**